### PR TITLE
When go to new page, stay on unless user turned off

### PR DESCRIPTION
**Pull request by benfrancis314**
I do this using a setting APP_STATUS. When user turns it on, this is set to true. When user turns it off, this is set to false. When running oneTimeSetup, before setting the click listener for R it checks this setting. If true, it turns on app. 

I think the usage of the settings is in accordance with the new setting strategy from PR #148, but let me know if I've misunderstood it.

### CODE REVIEW

File: extension/content_script.js [hunk 1]
```diff
@@ -584,13 +585,20 @@ function removeUI() {
...
+		settings.setAppStatus(true); 
 		setupUI();
 	} else {
 		removeUI();
+		settings.setAppStatus(false); 
 	}
 }
 
 function setupKeyListenerForOnOff() {
+	let appStatus = settings.getAppStatus();
```
Comment (sjonany) [inline extension/content_script.js]: What do you think about:
1. Move this appstatus snippet to oneTimeSetup.
 -- it's not really about setting up key listener, and this logic of "immediately show ui?" would be something i expected to see in oneTimeSetup.

2. im not sure why onetimesetup needs to accept a callback.
Why not just have setupKeyListenerForOnOff be called in line 552?
Then it's clear what methods are exactly being called once per webpage -- just read that one function. No need to figure out what cb() is.

**Review by sjonany — COMMENTED** [review]
setting approach looks good! V nice.
It's not the same as "all tabs are either all on or off", but this experience I'm fine with, and is maybe desirable.

**Comment by sjonany** [comment]
O just that 1 comment to resolve. code refactoring ish

File: extension/content_script.js (hunk 1 above)
Comment (benfrancis314) [reply extension/content_script.js]: Agree, agree

**Comment by sjonany** [comment]
LGTM

**Review by sjonany — APPROVED** [review]

**Pull request merged**